src/esp/nav/PathFinder.cpp

```diff
@@ -673,6 +702,32 @@ bool PathFinder::loadNavMesh(const std::string& path) {
 
   navMesh_ = mesh;
   bounds_ = std::make_pair(bmin, bmax);
+
+  // Some polygons have zero area for some reason.  When we navigate into a zero
+  // area polygon, things crash.  So we find all zero area polygons and mark
+  // them as disabled/not navigable.
+
+  // Iterate over all tiles
```

Comment:
Actually, can we break this out as a function so that I can also use it for recompute navmesh later?

Reply:
Done